src/main/app/src/app/admin/parameters/parameters.component.ts

```diff
@@ -33,7 +33,9 @@ export class ParametersComponent
   @ViewChild("parametersSort") parametersSort!: MatSort;
 
   filterParameters!: ZaakafhandelParametersListParameters;
-  parameters = new MatTableDataSource<ZaakafhandelParameters>();
+  parameters = new MatTableDataSource<
+    GeneratedType<"RestZaakafhandelParameters">
```

Comment:
Looks good but what does this have to do with betrokkene koppelen? Especially with "betrokkene koppelen - backend"?

Reply:
Seems to me there is too much different stuff in this PR..?